docs/configuration.asciidoc

```diff
@@ -315,6 +315,24 @@ To reduce overhead and storage requirements, you can set the sample rate to a va
 between `0.0` and `1.0`. We still record overall time and the result for unsampled
 transactions, but no context information, tags, or spans.
 
+[float]
+[[config-server-cert]]
+=== `ELASTIC_APM_SERVER_CERT`
+
+[options="header"]
+|============
+| Environment                | Default
+| `ELASTIC_APM_SERVER_CERT`  |
+|============
+
+If you have configured your APM Server with a self signed TLS certificate, you can
```

Comment:
would work even for an actual CA certified certificate right? I get that the primary use-case is self signed, but i've seen people pin them anyway. I don't feel strongly about rewording this, so feel free to leave it as is.

Reply:
I like that it says PEM encoded [further down](https://github.com/elastic/apm-agent-go/pull/325/files#diff-2d08121c527972df0fc40641b3ecc1a2R72). Might want to include that here as well.